src/app/application.tsx

```diff
@@ -51,7 +51,7 @@ import { EditorType } from "./stores/app_store";
 import { LocalizationConfig } from "../container/container";
 
 export class ApplicationExtensionContext implements ExtensionContext {
-  constructor(public app: Application) {}
+  constructor(public app: Application) { }
```

Comment:
I didn't do this. But I apparently opened this file in VSCode and some reformatting rule apparently insisted (it also rewrote my workflow yaml).

I'm willing to drop these changes, but do note that files should not trip up drive-by-contributors which means someone should really clean up the codebase so this doesn't happen when the next person drives by.